embedded-aerospike-enterprise/README.adoc

```diff
@@ -0,0 +1,65 @@
+=== embedded-aerospike-enterprise
+
+Extension for `embedded-aerospike` module.
```

Comment:
It's not an extension,  it's a full-blown container :)

Reply:
changed